### PR TITLE
Readded all changes

### PR DESCRIPTION
**Pull request by noahklowden**
This should simply reimplement the changes to the game events done after #104 so that it can be readded.

### CODE REVIEW

**Comment by gypark23** [comment]
Have you ran this on tns run ios? The app immediately closes when you launch.

**Comment by noahklowden** [comment]
> Have you ran this on tns run ios? The app immediately closes when you launch.
As I said in the meeting, I am not able to on my setup. I don't know which change seems to be causing the crash, then... Hm...

**Comment by Strongbrother1** [comment]
tns run ios crashes the program for me too, I'm not sure where though.

**Comment by gypark23** [comment]
> > Have you ran this on tns run ios? The app immediately closes when you launch.
> 
> As I said in the meeting, I am not able to on my setup. I don't know which change seems to be causing the crash, then... Hm...

The app closes right after the launch, so I am assuming that it is coming from the home component.ts. Though I am not really sure what is wrong with it...

**Comment by gypark23** [comment]
The new change didn't fix it FYI

**Comment by gypark23** [comment]
Still nope

File: appGeo/src/app/home/home.component.ts [hunk 1]
```diff
@@ -65,7 +67,9 @@ export class HomeComponent implements OnInit {
     // Init your component properties here.
     // Going to initialize a list of bubbles here;
     var map = new CampusMap();
+    databaseAdd(MAP_PATH, map)
     var gameRules = new GameRules();
+    databaseAdd(GAMERULE_PATH, gameRules)
```
[the remark this answers is not in the document]
Comment (gypark23) [reply appGeo/src/app/home/home.component.ts:81]: @noahklowden OK I just checked it, line 70 and line 72 are the source of errors. If you comment them out, the app runs.

**Review by gypark23 — CHANGES_REQUESTED** [review]

File: appGeo/src/app/home/home.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (noahklowden) [reply appGeo/src/app/home/home.component.ts:81]: Hm, alright. This is info that I need to pass through the database, though, so removing them is a temp fix, but... this won't work completely long term. Clearly I misinterpreted how the database works a bit

[the remark this answers is not in the document]
Comment (noahklowden) [reply appGeo/src/app/home/home.component.ts:81]: I'll test it a little more tonight. I've just about got tns run ios working on my laptop, finally

**Review by gypark23 — APPROVED** [review]

**Pull request merged**